src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/ProjectViewContainer.tsx

```diff
@@ -176,8 +177,8 @@ const ProjectViewContainer: React.FC<ProjectViewContainerProps> = ({ project, en
       >
         <EditButton />
         <CreateChangeRequestButton />
-        <SuggestBudgetIncreaseButton />
```

Comment:
Is there are reason you switched these around?

Reply:
If you try to add divider={true} to the "Unassign from My Team" button, only team leads will be able to properly see the implemented changes.

Reply:
Got it